packages/webapp/cypress/support/commands.ts

```diff
@@ -24,4 +24,18 @@
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
+Cypress.Commands.add("login", (account) => {
+    cy.get(`[data-testid="signin-nav-buttonsm"]`).first().click();
+    cy.wait(500);
+    cy.get("#ual-box div").contains("Password").click();
+    cy.wait(500);
+    cy.get('#ual-box input[type="text"]').type(account);
+    cy.wait(500);
+    cy.get("#ual-box div").contains("Continue").click();
+    cy.wait(500);
+    cy.get('input[type="password"]').type(Cypress.env("test_users_pk"));
+    cy.get('button[type="submit"]').click();
+    cy.wait(500);
```

Comment:
I remember seeing documentation about `wait` being an antipattern sometimes in the Cypress documentation. Just found it: https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

They also talk specifically in that section about alternatives to `cy.wait(arbitrary_time_period)` when used with `cy.get(...)`. 

I don't know whether the way you're doing it here is warranted or is the antipattern. But thought I'd mention it just in case.

Reply:
you are absolutely right when the waiting is unnecessary... I consciously added in two situations:
1. handle ual inputs, because it seemed like chained `cy.get()` didn't work...
2. wait for blocks to propagate after creating the induction, and this one was needed only for our CI pipeline because the headless browser runs super fast... 

That being said, let me go through this cypress reference to see if I have better ways of handling it!

Reply:
Adjusted! Can you please review it again?